compiler/src/yul/mappers/expressions.rs

```diff
@@ -38,11 +38,33 @@ pub fn expr(context: &Context, exp: &Spanned<fe::Expr>) -> Result<yul::Expressio
     }
 }
 
-pub fn expr_call(exp: &Spanned<fe::Expr>) -> Result<yul::Expression, CompileError> {
-    if let fe::Expr::Call { args: _, func } = &exp.node {
+pub fn expr_call(
+    context: &Context,
+    exp: &Spanned<fe::Expr>,
+) -> Result<yul::Expression, CompileError> {
+    if let fe::Expr::Call { args, func } = &exp.node {
         if let fe::Expr::Attribute { value: _, attr } = &func.node {
+            let arguments = &args.node;
+            let yul_args = arguments
+                .iter()
+                .map(|val| {
+                    if let fe::CallArg::Arg(arg_expr) = val.node.clone() {
+                        let spanned = Spanned {
+                            node: arg_expr,
+                            span: val.span,
+                        };
```

Comment:
Not sure if this is pointing to a bigger problem but I had to wrap this in a span, reusing the span around the `CallArg`.

Reply:
see: 

https://github.com/ethereum/fe/blob/703e31b5c0cf781f9cb10121b2a49d64896cbb19/semantics/src/traversal/expressions.rs#L84

This helper function exists in both the compiler and mapper libraries. I think we'll just need to accept using it in these situations.